### DISCUSSION TITLE
AttributeError: 'English' object has no attribute 'tag_map'

### DISCUSSION THREAD

**Discussion by vranjan2** [explosion/spaCy#9101] [Help: Coding & Implementations] [answered]
I am trying to run a code hosted at: https://github.com/andreasvc/spacyconllu/blob/master/spacyconllu.py

This 
nlp = spacy.load("en_core_web_sm")
tagmap = nlp.Defaults.tag_map

I get an error:
AttributeError: 'English' object has no attribute 'tag_map'

How do I get tag_map in the latest spacy?

**Comment by polm** [comment, marked as answer]
There is no `tag_map` in v3. 

It looks like the code you are trying to use is not maintained. If you want to use it you can use v2. 

Alternately, generating CoNLLu data isn't that hard, so you should be able to just loop over tokens and print what you need.

**Comment by BramVanroy** [comment]
Hi there. The code over at andreasvc is an old fork of [my spacy_conll library](https://github.com/BramVanroy/spacy_conll) that is not up-to-date with upstream. You should be able to use mine without issue on spaCy v3.

EDIT; I am wrong, apparently it is a sister-fork of the original work with rgalhama. The project has evolved quite a bit since then.
